scripts/mockdata/scenarios.json

```diff
@@ -8,7 +8,20 @@
     "is_previewable": true,
     "publish_link": null,
     "preview_link": "www.previewlink.com/student-escape-room",
-    "expected_solve_time": "10 to 20"
+    "expected_solve_time": "10 to 20",
+    "transitions": [
+      {
+        "data": [
+          { "text": "text and an image", "imageSrc": "bananaaaaaaaaaa" },
```

Comment:
Valid imageSrc 🙏  so that we can integrate easily. So similar to the other objects, something like `/static/img/lightOn.png`